Documentation/updateUSERGUIDE-Cordio.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+from os import listdir
+
```

Comment:
So this looks for markdown files and PDFs and adds links to USERGUIDE.md?

Maybe we should move this to the workflows folder?

Reply:
moved, but we will have to do a test to double check the new pathing from that file